addons/hr_org_chart/static/src/js/hr_org_chart.js

```diff
@@ -175,10 +184,10 @@ var FieldOrgChart = AbstractField.extend({
                     view_mode: 'kanban,list,form',
                     views: [[false, 'kanban'], [false, 'list'], [false, 'form']],
                     target: 'current',
-                    res_model: 'hr.employee',
+                    res_model: this.model,
```

Comment:
Instead of adding a useless route `/hr/get_redirect_model` just for this, why don't we call `get_formview_action` and then call `do_action` on the result. `get_formview_action` already contains this magic.